resources/geometry.rb

```diff
@@ -61,6 +124,28 @@ def self.get_height_of_spaces(spaces)
     end
     return maxzs.max - minzs.min
   end
+  
+  def self.get_height_of_thermal_zone(building:,
+                                      thermal_zone:)
+    heights = []
+    building.elements.each("BuildingDetails/Enclosure/FoundationWalls/FoundationWall") do |foundation_wall|
+      foundation_wall_values = HPXML.get_foundation_wall_values(foundation_wall: foundation_wall)
+      next if foundation_wall_values[:interior_adjacent_to] != thermal_zone
+
+      height = foundation_wall_values[:height]
+      if building.elements["BuildingDetails/Enclosure/RimJoists/RimJoist"]
+        height += 1.0
```

Comment:
Need to find out a way to avoid making similar assumptions different places in the code.

Reply:
`Constants.RimJoistHeight`?